packages/app/src/cli/models/extensions/specifications/flow_template.ts

```diff
@@ -0,0 +1,83 @@
+// import {createUIExtensionSpecification} from '../ui.js'
+import {BaseSchema} from '../schemas.js'
+import {createExtensionSpecification} from '../specification.js'
+import {loadLocalesConfig} from '../../../utilities/extensions/locales-configuration.js'
+import {joinPath} from '@shopify/cli-kit/node/path'
+import {zod} from '@shopify/cli-kit/node/schema'
+import {AbortError} from '@shopify/cli-kit/node/error'
+import {glob} from '@shopify/cli-kit/node/fs'
+import fs from 'fs'
+
+const FlowTemplateExtensionSchema = BaseSchema.extend({
+  name: zod.string(),
+  description: zod.string(),
+  extensions: zod.array(
+    zod.object({
+      handle: zod.string(),
+      type: zod.literal('flow_template'),
+      title: zod.string(),
+      description: zod.string(),
+      categories: zod.array(zod.string()),
+      pre_install_note: zod.string(),
+      post_install_note: zod.string(),
+      require_app: zod.boolean(),
+      visible: zod.boolean(),
+      enabled: zod.boolean(),
+    }),
+  ),
```

Comment:
add `.min(1)` here so that `zod` will take care of validating that at least 1 extension exists. You can also say `max(1)`I think.